### ISSUE TITLE
Ability to Sort the data axis

### ISSUE THREAD

**Issue by ruthpozuelo** [microsoft/charticulator#807]
**Is your feature request related to a problem? Please describe.**
I need to be able to sort the data axis.
I understand that I can reverse the range, but that seems to hardcode the values and it does not work in Power BI:
![image](https://user-images.githubusercontent.com/19476550/138649726-89696944-00a5-4534-b8a5-692b11f2fffb.png)
![image](https://user-images.githubusercontent.com/19476550/138649744-6718b0ee-c482-471a-b5b8-5a0c9e3037ae.png)

**Comment by dm-p** [comment]
Hi Ruth - have you tried drawing the data axis from top to bottom instead of bottom to top? This usually works for me and lets you keep the range in the right order:

![image](https://user-images.githubusercontent.com/10572054/141841272-b1af34f1-223c-404a-9411-4b32ae7dc3c2.png)

![image](https://user-images.githubusercontent.com/10572054/141841373-9024c571-4dab-4c99-acfb-11555b0926c0.png)

In Power BI:

![image](https://user-images.githubusercontent.com/10572054/141841160-c6e9c81b-ee34-4759-aaf9-831a96c22cfc.png)

**Comment by ruthpozuelo** [comment]
I havent , but I will try, Thanks!
I am still requesting a sort feature on the UI :)

**Comment by dm-p** [comment]
Yup - the sort makes sense as a feature request, as this is not intuitive, but it may help you for now

**Comment by ruthpozuelo** [comment]
You are a ⭐️, thank you Daniel!
